src/constructors.jl

```diff
@@ -98,6 +102,10 @@ E.g. 1, 2f0, 4.0
     return :($FSA(a))
 end
 
+@generated function call{FSV <: FixedVectorNoTuple, N}(::Type{FSV}, a::FixedVector{N})
+    return Expr(:call, FSV, ntuple(i->:(a[$i]), N)...)
```

Comment:
Should we have a size check here to make sure `a` has the right number of elements?